pytest/test_sessions.py

```diff
@@ -198,6 +198,30 @@ def testCommandReaderRegisterSameTrigger(env):
     '''
     env.expect('RG.PYEXECUTE', script, 'ID', 'test', 'DESCRIPTION', 'desc').error().contains('trigger already registered in this session')
 
+@gearsTest(skipOnCluster=True)
+def testMod4054(env):
+    script = '''
```

Comment:
It's small, and silly - but I'd triple quote with double qutoes (use " in lines 203/222)